### DISCUSSION TITLE
Why isn't the user-defined zoom scale option within my settings taking effect？

### DISCUSSION THREAD

**Discussion by linjiangang123** [upscayl/upscayl#765] [Q&A]
![image](https://github.com/upscayl/upscayl/assets/165828079/c57bb6f1-ac35-48ab-a124-3123e070e218)
i changed custom output width is 3584，but output image is origin width*3
![image](https://github.com/upscayl/upscayl/assets/165828079/5593e92a-b854-4df6-8dbf-d5d90dc49da2)

**Comment by NayamAmarshe** [comment]
We'll be updating this soon.
